django/db/backends/sqlite3/introspection.py

```diff
@@ -316,8 +316,8 @@ def get_constraints(self, cursor, table_name):
         # Find inline check constraints.
         try:
             table_schema = cursor.execute(
-                "SELECT sql FROM sqlite_master WHERE type='table' and name=%s"
-                % (self.connection.ops.quote_name(table_name),)
+                "SELECT sql FROM sqlite_master WHERE type='table' and name=%s",
+                (table_name,),
```

Comment:
```suggestion
                [table_name],
```

Reply:
I've checked "introspection" modules in other backends and we always pass `params` as a list :thinking: There are a few places in the SQLite backend when we use tuples but I don't think it's worth changing them per se.

Reply:
It's only a suggestion :wink: